cluster-api-1.9.yaml

```diff
@@ -0,0 +1,143 @@
+package:
+  name: cluster-api-1.9
+  version: "1.9.6"
+  epoch: 0
+  description: Cluster API meta package
+  dependencies:
+    provides:
+      - cluster-api=${{package.full-version}}
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 813e90d8daef60b715764ce4729b40d82f811cc1
+      repository: https://github.com/kubernetes-sigs/cluster-api
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
+
+subpackages:
```

Comment:
Can we use `item:` pipeline here? And iterate over packages in the single subpkg. `nvidia-container-toolkit` comes mind as an example.

Reply:
item doesn't work a 100% but added where useful